Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -193,7 +193,7 @@ public class FieldNpc : Actor<Npc> {
 
         if (SendControl && !IsDead) {
```

Comment:
Control does not stop sending until the NPC is removed. It's sent even when it's dead

Reply:
Oops, gonna fix that